content/md/en/docs/tutorials/build-a-blockchain/authorize-specific-nodes.md

```diff
@@ -131,7 +131,7 @@ To add the `node-authorization` pallet to the Substrate runtime:
 
    ```toml
    [dependencies]
-   pallet-node-authorization = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+   pallet-node-authorization = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0" }
```

Comment:
Again `branch`

Reply:
The new repo should be available in 2 weeks, clarified this with Sacha. Will change it after the repo is available, and till then we can go ahead with this.